spec/views/events/show.html.erb_spec.rb

```diff
@@ -96,4 +89,78 @@
+    expect(rendered).to_not have_link(t(:print_all, scope: 'events.applicants_overview'))
+    expect(rendered).to_not have_link(t(:accept_all, scope: 'events.applicants_overview'))
+    expect(rendered).to_not have_link(t(:sending_acceptances, scope: 'events.applicants_overview'))
+    expect(rendered).to_not have_link(t(:sending_rejections, scope: 'events.applicants_overview'))
+    expect(rendered).to_not have_link(t(:show_participants, scope: 'events.participants'))
+    expect(rendered).to_not have_button(t(:sending_acceptances, scope: 'events.applicants_overview'), disabled: true)
+    expect(rendered).to_not have_button(t(:sending_rejections, scope: 'events.applicants_overview'), disabled: true)
+  end
+
+  it "displays correct buttons in application phase" do
+    @event = assign(:event, FactoryGirl.create(:event, :in_application_phase))
+    sign_in(FactoryGirl.create(:user, role: :organizer))
+    render
+    expect(rendered).to_not have_link(t(:print_all, scope: 'events.applicants_overview'))
+    expect(rendered).to_not have_link(t(:accept_all, scope: 'events.applicants_overview'))
+    expect(rendered).to_not have_link(t(:sending_acceptances, scope: 'events.applicants_overview'))
+    expect(rendered).to_not have_link(t(:sending_rejections, scope: 'events.applicants_overview'))
+    expect(rendered).to_not have_link(t(:show_participants, scope: 'events.participants'))
+    expect(rendered).to_not have_button(t(:sending_acceptances, scope: 'events.applicants_overview'), disabled: true)
+    expect(rendered).to_not have_button(t(:sending_rejections, scope: 'events.applicants_overview'), disabled: true)
+  end
+
+  it "does not display the disabled send email buttons in application phase (even when there are unclassified applications)" do
+    @event = assign(:event, FactoryGirl.create(:event, :with_diverse_open_applications, :in_application_phase))
+    sign_in(FactoryGirl.create(:user, role: :organizer))
+    render
+    expect(rendered).to_not have_button(t(:sending_acceptances, scope: 'events.applicants_overview'), disabled: true)
+    expect(rendered).to_not have_button(t(:sending_rejections, scope: 'events.applicants_overview'), disabled: true)
+  end
+
+  it "displays correct buttons in selection phase" do
+    @event = assign(:event, FactoryGirl.create(:event, :in_selection_phase))
+    sign_in(FactoryGirl.create(:user, role: :organizer))
+    render
+    expect(rendered).to have_link(t(:print_all, scope: 'events.applicants_overview'))
+    expect(rendered).to have_link(t(:accept_all, scope: 'events.applicants_overview'))
+    expect(rendered).to have_link(t(:sending_acceptances, scope: 'events.applicants_overview'))
+    expect(rendered).to have_link(t(:sending_rejections, scope: 'events.applicants_overview'))
+    expect(rendered).to_not have_link(t(:show_participants, scope: 'events.participants'))
+  end
+
+  it "does display the disabled send email buttons in selection phase (when there are unclassified applications)" do
```

Comment:
Just for consistency. In the other cases it is "it displays..." instead of "it does display...."